cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -350,14 +350,14 @@ function DownloadAndInstall-AuthPlugin {
   }
 
   if (-not ($kube_env.ContainsKey('EXEC_AUTH_PLUGIN_LICENSE_URL') -and
-            $kube_env.ContainsKey('EXEC_AUTH_PLUGIN_SHA1') -and
+            $kube_env.ContainsKey('EXEC_AUTH_PLUGIN_HASH') -and
```

Comment:
is there going to be a separate cl to populate kube-env with EXEC_AUTH_PLUGIN_HASH? currently EXEC_AUTH_PLUGIN_SHA1 is in kube-env, but EXEC_AUTH_PLUGIN_HASH is not.

Reply:
yes